src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Controllers/ManageEvidenceNotesControllerReviewEvidenceNoteTests.cs

```diff
@@ -291,41 +314,30 @@ public async Task ReviewEvidenceNotePost_GivenInvalidModel_BreadcrumbShouldBeSet
             Breadcrumb.OrganisationId.Should().Be(organisationId);
         }
 
-        [Fact]
-        public async Task ReviewEvidenceNotePost_GivenInvalidModel_ModelShouldBeReturned()
-        {
-            //arrange
-            var httpContext = new HttpContextMocker();
-            var model = Fixture.Create<ReviewEvidenceNoteViewModel>();
-            AddModelError();
-            EvidenceNoteData note = Fixture.Create<EvidenceNoteData>();
-            A.CallTo(() => WeeeClient.SendAsync(A<string>._, A<GetEvidenceNoteForSchemeRequest>._)).Returns(note);
-            httpContext.AttachToController(ManageEvidenceController);
-
-            //act
-            var result = await ManageEvidenceController.ReviewEvidenceNote(model) as ViewResult;
-
-            //assert
-            result.Model.Should().NotBeNull();
-        }
-
         [Fact]
         public async Task ReviewEvidenceNotePost_GivenAModelIsValid_ShouldRedirectToDownloadEvidenceNoteAction()
         {
             //arrange
-            var httpContext = new HttpContextMocker();
-            httpContext.AttachToController(ManageEvidenceController);
+            var organisationId = Fixture.Create<Guid>();
+            var evidenceNoteId = Fixture.Create<Guid>();
+            var complianceYear = Fixture.Create<int>();
             var model = GetValidModel();
+            model.ViewEvidenceNoteViewModel.OrganisationId = organisationId;
+            model.ViewEvidenceNoteViewModel.Id = evidenceNoteId;
+            model.ViewEvidenceNoteViewModel.SelectedComplianceYear = complianceYear;
```

Comment:
kinda related to compliance year, there is this ComplianceYearDisplay in ViewEvidenceNoteViewModel class. which gets StartDate. Does it need to be updated to get the SelectedComplianceYear ?